Makefile

```diff
@@ -1483,12 +1483,12 @@ tcpto.h readwrite.h timeoutconn.h timeoutread.h timeoutwrite.h
 
 qmail-rspawn: \
 load qmail-rspawn.o spawn.o tcpto_clean.o now.o coe.o sig.a open.a \
-seek.a lock.a wait.a fd.a stralloc.a alloc.a substdio.a error.a env.a str.a \
```

Comment:
This dependency change is wrong. Just use env.a for the environment functions, and drop explicit str_diffn.o as these function is already dragged in via str.a.

Reply:
Fixed the dependency. Had to put str.a after env.a to fix unresolved symbol for str_diffn()